packages/docusaurus-theme-classic/src/theme/Blog/Pages/BlogAuthorsPostsPage/index.tsx

```diff
@@ -52,9 +53,21 @@ function Content({author, items, sidebar, listMetadata}: Props): JSX.Element {
         {author.description && <p>{author.description}</p>}
         <ViewAllAuthorsLink />
       </header>
-      <hr />
-      <BlogPostItems items={items} />
-      <BlogListPaginator metadata={listMetadata} />
+      {items.length === 0 ? (
+        <p>
+          <Translate
+            id="theme.BlogAuthorsPostsPage.Content.noPosts"
+            description="The text for authors with 0 blog post">
+            This author has not written any posts yet.
+          </Translate>
```

Comment:
```suggestion
          <Translate
            id="theme.blog.author.noPosts"
            description="The text for authors with 0 blog post">
            This author hasn't written any posts yet.
          </Translate>
```

minor changes + move to blogTranslations.ts